ld_script_modern.ld

```diff
@@ -83,9 +77,7 @@ SECTIONS {
     .rodata :
     ALIGN(4)
     {
-        src/*.o(.rodata*);
-        gflib/*.o(.rodata*);
-        data/*.o(.rodata*);
+        *(.rodata*)
```

Comment:
How come you've written this as `*(.rodata*)` instead of `*(.rodata)` and `*(.rodata.*)` like you did with `.text`/`.bss`?